Modules/Registration/PDEDeformable/wrapping/itkSymmetricForcesDemonsRegistrationFilter.wrap

```diff
@@ -1,7 +1,18 @@
 # DemonsRegistrationFilter is wrapped with POINTER_WITH_SUPERCLASS
 # so this class is not.
 itk_wrap_class("itk::SymmetricForcesDemonsRegistrationFilter" POINTER)
-  foreach(s ${WRAP_ITK_SCALAR})
-    itk_wrap_image_filter_combinations("${s}" "${s}" "${WRAP_ITK_VECTOR_REAL}" 2+)
+  foreach(curr_dim ${ITK_WRAP_IMAGE_DIMS})
+    if(${curr_dim} LESS 1)
+        continue()
+    endif()
+    # Vector components dimension must match image dimension. Temporarily change the global vector dims list.
+    set(BACKUP_ITK_WRAP_VECTOR_COMPONENTS "${ITK_WRAP_VECTOR_COMPONENTS}")
```

Comment:
... replace `BACKUP_` ....

Reply:
A few lines later `BACKUP_*` gets unset, thus deleting the definition. I don't know what would be the result if `BACKUP_*` definition existed prior to this code. It would probably be undefined, but potential for problems is not that big.